src/platforms/javascript/common/performance.mdx

```diff
@@ -296,3 +288,8 @@ span.setData("parameters", parameters);
 span.setData("user_id", parameters.user_id);
 http.get(`${base_url}/${endpoint}/`, (data = parameters));
 ```
+
```

Comment:
We've removed the Next Steps links that are hand managed in the SDK content - the goal will be to replace them with something that's auto-generated.